v2/community/database-setup/mysql.mdx

```diff
@@ -107,9 +107,9 @@ CREATE TABLE IF NOT EXISTS all_auth_recipe_users(
 CREATE INDEX all_auth_recipe_users_pagination_index ON all_auth_recipe_users (time_joined DESC, user_id DESC);
 
 CREATE TABLE session_access_token_signing_keys (
-  created_at_time bigint unsigned NOT NULL,
-  value text,
-  PRIMARY KEY (created_at_time)
+    created_at_time bigint unsigned NOT NULL,
+    value text,
+    PRIMARY KEY (created_at_time)
```

Comment:
- Other tables have use capital letters for `bigint`, `unsigned `, and `text`. Please make them caps.
- Also, how come there are no changes to the other files that use this? Did you not run `npm run start` when you made these changes?